internal/tiltfile/k8s.go

```diff
@@ -111,17 +111,34 @@ func (r k8sResource) refSelectorList() []string {
 
 func (s *tiltfileState) k8sYaml(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
 	var yamlValue starlark.Value
+
 	if err := s.unpackArgs(fn.Name(), args, kwargs,
 		"yaml", &yamlValue,
 	); err != nil {
 		return nil, err
 	}
+	//normalize the starlark value into a slice
+	value := starlarkValueOrSequenceToSlice(yamlValue)
 
-	entities, err := s.yamlEntitiesFromSkylarkValueOrList(thread, yamlValue)
-	if err != nil {
-		return nil, err
+	//if `None` was passed into k8s_yaml, len(val) = 0
+	if len(value) > 0 {
+
+		val, _ := starlark.AsString(value[0])
+		entities, err := s.yamlEntitiesFromSkylarkValueOrList(thread, yamlValue)
+
+		if err != nil {
+			return nil, err
+		}
+
+		//the parameter blob('') results in an empty string
+		if len(entities) == 0 && val == "" {
+			return nil, fmt.Errorf("k8s_yaml: Empty or Invalid YAML Resource Detected")
```

Comment:
i don't think you need the `k8s_yaml: ` prefix? the starlark infrastructure will already print the function name for you

Reply:
Oh right, yeah it does : 
![Screenshot from 2020-06-18 17-25-25](https://user-images.githubusercontent.com/35721712/85025709-cb23a180-b188-11ea-907e-26e6fbbcc308.png)